### ISSUE TITLE
Redis administration

### ISSUE THREAD

**Issue by loiclavoie** [redis/redis-doc#1937]
The document in Redis Administration provides an incorrect command for disabling "transparent huge pages" on Linux Kernel.

It current states:
> ...This is accomplished with the following command: echo madvise > /sys/kernel/mm/transparent_hugepage/enabled.

The command should be
`echo never > /sys/kernel/mm/transparent_hugepage/enabled`

Source redhat: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/performance_tuning_guide/sect-red_hat_enterprise_linux-performance_tuning_guide-configuring_transparent_huge_pages

**Comment by juckerf** [comment]
@banker during some research about the "disable THP" recommendation I stumbled upon this PR. In https://github.com/redis/redis-doc/pull/841 the recommendation was changed from `never` to `madvise` (based on this https://github.com/redis/redis/pull/4001/files) but now reverted in this PR - what's the recommendation now?

**Comment by banker** [comment]
@juckerf Sorry for the confusion here. @oranagra or @itamarhaber: can you advise?

**Comment by oranagra** [comment]
@banker, i agree, THP should be set to `madvise` (as is the default in some platforms), see https://github.com/redis/redis/pull/7771
maybe we can mention that it should be `never` if you wanna silence warnings in redis versions prior to 6.2.0 and 6.0.8.

@juckerf thanks for catching this!
